front/src/Components/UI/ErrorScreen.svelte

```diff
@@ -0,0 +1,157 @@
+        <p>{$errorScreenStore.subtitle}</p>
+        {#if $errorScreenStore.type !== "retry"}<p class="code">Code : {$errorScreenStore.code}</p>{/if}
+        <p class="details">
+            {detailsStylized}{#if $errorScreenStore.type === "retry"}<div class="loading" />{/if}
+        </p>
+        {#if ($errorScreenStore.type === "retry" && $errorScreenStore.canRetryManual) || ($errorScreenStore.type === "redirect" && (window.history.length > 2 || $errorScreenStore.urlToRedirect))}
+            <div class="button" on:click={click}>
+                <img src={$errorScreenStore.type === "retry" ? reload : external} alt="" class="reload" />
+                {$errorScreenStore.buttonTitle}
+            </div>
+        {/if}
+    </div>
+</main>
+
+<style lang="scss">
+    main.errorScreen {
+        pointer-events: auto;
+        width: 100%;
+        background-color: #000000;
+        color: #ffffff;
+        text-align: center;
+        position: absolute;
+        display: flex;
+        flex-wrap: wrap;
+        align-items: center;
+        justify-content: center;
+        height: 100%;
+        top: 0;
+        left: 0;
+        right: 0;
+        margin-left: auto;
+        margin-right: auto;
+        min-width: 300px;
+        z-index: 700;
+        overflow-y: scroll;
+        padding: 20px 0;
+        .logo {
+            width: 50%;
+            margin-bottom: 50px;
+        }
+        .icon {
+            height: 125px;
+            margin-bottom: 25px;
+        }
+        h2 {
+            font-family: "Press Start 2P";
+            padding: 5px;
+            font-size: 30px;
+        }
+        p {
+            font-family: "Press Start 2P";
+        }
+        p.code {
+            font-size: 12px;
+            opacity: 0.6;
+            user-select: text;
+        }
+        p.details {
+            font-size: 12px;
+            max-width: 80%;
+            margin: 0 auto 35px auto;
+        }
+        .loading {
+            display: inline-block;
+            min-width: 20px;
+            position: relative;
+            margin-left: 5px;
+        }
+        .loading:after {
+            overflow: hidden;
+            display: inline-block;
+            vertical-align: bottom;
+            -webkit-animation: ellipsis steps(4, end) 900ms infinite;
+            animation: ellipsis steps(4, end) 900ms infinite;
+            content: "\2026";
+            width: 0;
+            font-family: "Press Start 2P";
+            font-size: 16px;
+            position: absolute;
+            left: 0;
+            top: -19px;
+        }
+
+        @keyframes ellipsis {
+            to {
+                width: 1.25em;
+            }
+        }
+
+        @-webkit-keyframes ellipsis {
+            to {
+                width: 1.25em;
+            }
+        }
+
+        .button {
+            cursor: pointer;
+            background-image: url("../images/button-large.png");
```

Comment:
We have a class you can use instead of a button.
We use NES.CSS => https://nostalgic-css.github.io/NES.css/
So you can simply use `class="nes-btn"` (and potentially change the background color if needed)